ddtrace/internal/compat.py

```diff
@@ -251,3 +251,7 @@ def maybe_stringify(obj):
     if obj is not None:
         return stringify(obj)
     return None
+
+
+def current_thread_ident():
+    return threading.current_thread().ident
```

Comment:
this `compat` module is for compat between Python 2 and 3 (mostly). I don't see any "compat" code here…